backend-app/tests/e2e/auth/login_signup.test.js

```diff
@@ -0,0 +1,88 @@
+const request = require('supertest');
+const app = require('../../../app');
+const { testUserCredentials } = require('../../testConstants');
+const { createUser, deleteUser } = require('../../utils');
+const { REQUIRE_ACTIVATION } = require('../../../config/app_config');
+
+describe('POST /api/auth/login', () => {
+    // console.log('db connected: ', mongoose.connection.readyState);
+
+    beforeAll(async () => {
+        await createUser(
+            app,
+            testUserCredentials.userName,
+            testUserCredentials.userEmail,
+            testUserCredentials.userPassword
+        );
```

Comment:
We can use database seeding. What do you think?

Reply:
yes it is a great idea i already created seedData.js, you can start with that and create an issue for it so we can work on it 
@muttaqin1